src/python_testing/TC_TLSCERT.py

```diff
@@ -86,7 +108,7 @@ def gen_cert(self) -> bytes:
             CertificateBuilder()
             .subject_name(root_cert_subject)
```

Comment:
this should come from the csr.

Reply:
Thank you, I wasn't sure what I was supposed to stuff from the CSR & figured it probably didn't matter too much as the key is the important part 🤣 fixed